Source/Core/Core/IOS/ES/ES.cpp

```diff
@@ -242,6 +242,23 @@ bool ES::LaunchTitle(u64 title_id, bool skip_reload)
   // (supposedly when trying to re-open those files).
   DiscIO::NANDContentManager::Access().ClearCache();
 
+  u32 device_id;
+  constexpr u32 DEFAULT_DEVICE_ID = 0x0403ac68;
+  if (GetDeviceId(&device_id) != IPC_SUCCESS ||
+      (title_id == Titles::SHOP && device_id == DEFAULT_DEVICE_ID))
```

Comment:
Don't you want to check `title_id == Titles::SHOP` before checking `GetDeviceId(&device_id) != IPC_SUCCESS`, so that there never will an error message about the shop channel when the user is launching an unrelated channel?